### DISCUSSION TITLE
CAT remote control

### DISCUSSION THREAD

**Discussion by HeLyO** [gdyuldin/x6100_gui#42] [Q&A] [answered]
Hi,
Did you foreseen to implement all CAT instructions in a future version of the firmware?
I see that CAT instructions are listed in the file cat.c , but not all are yet functional (NR, NF, DNF, ...).
Thanks

**Comment by gdyuldin** [comment, marked as answer]
Hi. I'm not going to add another CAT commands, at least right now. My goal was to be able to use this radio with WSJT-X and HDSDR.
Perhaps, a bit later.
